chia/simulator/SimulatorFullNodeRpcApi.py

```diff
@@ -9,12 +9,30 @@
 class SimulatorFullNodeRpcApi(FullNodeRpcApi):
     def get_routes(self) -> Dict[str, Endpoint]:
         routes = super().get_routes()
-        routes["/farm_tx_block"] = self.farm_tx_block
+        routes["/farm_block"] = self.farm_block
+        routes["/auto_farm"] = self.auto_farm
         return routes
 
-    async def farm_tx_block(self, _request: Dict[str, object]) -> Dict[str, object]:
+    async def farm_block(self, _request: Dict[str, object]) -> Dict[str, object]:
         request_address = str(_request["address"])
```

Comment:
I would instead provide an optional parameter: `guarantee_tx_block` which defaults to False. Also, FullBlock can be a tx block. We can never guarantee a non-tx block.  So if `guarantee_tx_block` is true, then we force a tx block to be created. Otherwise, we just create a block but we don't know whether it is tx or not.